### ISSUE TITLE
Minimum image size/resolution/dpi

### ISSUE THREAD

**Issue by hillbicks** [OlafenwaMoses/DeepStack_OpenLogo#3]
Hey there,

first of all, thanks for putting together the data set!

I setup the custom model yesterday and first tests with logos downloaded directly from the internet worked fine with the model. I was able to get the correct response. 

However, I want to use images from my doorbell camera and analyse the pictures with cars in it (detection done by frigate) for any logos. The images I used were 1920x1080 but only 96dpi. The fedex sample image meanwhile has 400dpi. 

Do you have any experience what the minimum "quality" for the source image should be?

**Comment by hillbicks** [comment]
@OlafenwaMoses I would be really thankful if you could give me a hint in the right direction for this :)

Thanks!

**Comment by hillbicks** [comment]
I got an answer from another maintainer which pointed me in the right direction. For anybody interested, here is the link. 

https://github.com/sstratoti/DeepStack_USPS/issues/1

tldr: apparently it is not the resolution, but the size of the object in the picture. Once I cropped out the delivery truck from the picture, deepstack was able to recognize the logo, although the image quality was far worse than the original.
